test/foundry/deploy/10_holograph_tests.t.sol

```diff
@@ -2,698 +2,759 @@
+   * @dev This test is designed to verify that the contract reverts as expected when the contract has already been initialized.
+   * This test expects a revert with the message 'HOLOGRAPH: already initialized' when trying to initialize
+   * a Holograph contract that is already initialized.
+   * Refers to the hardhat test with the description 'should fail to init if already initialized'
+   */
+  function testInitAlreadyInitializedRevert() public {
+    vm.expectRevert(bytes(ErrorConstants.ALREADY_INITIALIZED_ERROR_MSG));
+    holograph.init(initCode);
+  }
+
+  /* -------------------------------------------------------------------------- */
+  /*                                 GET BRIDGE                                 */
+  /* -------------------------------------------------------------------------- */
+
+  /**
+   * @notice Test the validity of the bridge slot in the Holograph contract.
+   * @dev This test is designed to ensure the correct functionality of the bridge slot in the contract.
+   * This test verifies that the value returned by `getBridge()` in the Holograph contract
+   * matches the expected `bridge` address.
+   * Refers to the hardhat test with the description 'Should return valid _bridgeSlot'
+   */
+  function testReturnValid_bridgeSlot() public view {
+    assertEq(holograph.getBridge(), bridge);
+  }
+
+  /**
+   * @notice Test the ability of an external contract to call the getBridge function in the Holograph contract.
+   * @dev This test is designed to verify that an external contract can successfully call the getBridge
+   * function in the Holograph contract.
+   * This test encodes the signature of the getBridge function and calls it from an external contract
+   * using mockExternalCall.
+   * Refers to the hardhat test with the description 'Should allow external contract to call fn'
+   */
+  function testAllowExternalContractToCallGetBridge() public {
+    bytes memory encodeSignature = abi.encodeWithSignature("getBridge()");
+    mockExternalCall.callExternalFn(address(holograph), encodeSignature);
+  }
+
+  /* -------------------------------------------------------------------------- */
+  /*                                 SET BRIDGE                                 */
+  /* -------------------------------------------------------------------------- */
+
+  /**
+   * @notice Test the ability of the admin to alter the bridge slot in the Holograph contract.
+   * @dev This test is designed to verify that the admin can alter the bridge slot in the contract.
+   * This test performs a prank operation on the current admin address of the Holograph contract,
+   * and then sets a new random address as the bridge in the contract.
+   * Refers to the hardhat test with the description 'should allow admin to alter _bridgeSlot'
+   */
+  function testAllowAdminAlter_bridgeSlot() public {
+    address random = RandomAddress.randomAddress();
+    vm.prank(origin);
+    holograph.setBridge(random);
```

Comment:
same here, im callint to current bridge to verify the correct set

Reply:
Done